modules/system-source/system-source.c

```diff
@@ -304,6 +304,10 @@ system_generate_system_transports(GString *sysblock)
     {
       system_sysblock_add_unix_dgram(sysblock, "/dev/log", NULL, NULL);
     }
+  else if (strcmp(u.sysname, "OpenBSD") == 0)
+    {
+      g_string_append_printf(sysblock, "opensd(\"%s\");", "/dev/klog");
```

Comment:
I think there's a typo there, that `opensd` should likely be `openbsd`

Reply:
typo fixed